### DISCUSSION TITLE
[新功能需求] 轮播图能不能把图片设置成插槽，不仅仅是图片，普通的div也是可以的

### DISCUSSION THREAD

**Discussion by SunHengX** [Moonofweisheng/wot-design-uni#89] [Ideas]
### 这个功能解决了什么问题？

类似于驾考宝典,可以左右滑动来切换题目

### 你期望的 API 是什么样子的？

类似于驾考宝典,可以左右滑动来切换题目

**Comment by Moonofweisheng** [comment]
这样的话需要将swiper-item提供出来，将会是一个破坏性的更新，我想这个问题可能要在以后再处理了。

**Comment by SunHengX** [comment]
> 这样的话需要将swiper-item提供出来，将会是一个破坏性的更新，我想这个问题可能要在以后再处理了。

好的,我临时是这样修改的
![image](https://github.com/Moonofweisheng/wot-design-uni/assets/63703285/05173ce6-eff0-4a58-98d9-5a5033ac940c)

**Comment by Moonofweisheng** [comment]
> > 这样的话需要将swiper-item提供出来，将会是一个破坏性的更新，我想这个问题可能要在以后再处理了。
> 
> 好的,我临时是这样修改的 ![image](https://user-images.githubusercontent.com/63703285/276471884-05173ce6-eff0-4a58-98d9-5a5033ac940c.png)

这样写也是可以的，可以先这样做，我会把这个功能加入开发计划中。

**Comment by Moonofweisheng** [comment]
见[2024开发计划](https://github.com/users/Moonofweisheng/projects/1)

**Comment by daiyuanhao** [comment]
需要这个功能+1
